include/hip/hip_runtime_api.h

```diff
@@ -126,6 +126,10 @@ typedef struct hipDeviceProp_t {
     size_t textureAlignment;         ///<Alignment requirement for textures
     int kernelExecTimeoutEnabled;    ///<Run time limit for kernels executed on the device
     int ECCEnabled;                  ///<Device has ECC support enabled
+#ifdef __HIP_PLATFORM_NVCC__
+    int tccDriver;                   ///< If device is Tesla device using TCC driver
+#endif
+
```

Comment:
We may leave this field in HIP stack as well for compatibility but report 0.

Reply:
This should not be guarded. Otherwise code will not compile on both platforms. We should set this to field to 0 on AMD path.